src/op_scenarios/opscenarios.jl

```diff
@@ -14,6 +14,24 @@ function probability(scen::AbstractOperationalScenario)
     return error("probabilty not implemented for type $(typeof(scen))")
 end
 
+"""
+    probability_scen(scen)
+
+The probability of a single scenario in a set of operational scenarios.
+"""
+probability_scen(scen::AbstractOperationalScenario) = probability(scen)
```

Comment:
This may lead to issues for the `TwoLevelTree` structure, if we do not explicitly provide a method. You did this for `StratNodeReprOpScenario`, but not `StratNodeOpScenario`. Hence, it would result in wrong values for `StratNodeOpScenario`.

Reply:
Fixed and tests added